### PR TITLE
added route for exam managementand created basic layout including but…

### PR DESCRIPTION
**Pull request by scottrumsby**
added route for exam management and created basic layout including button controls and table

### CODE REVIEW

File: frontend/src/layout/nav.vue [hunk 1]
```diff
@@ -77,7 +77,11 @@
       showAdmin() {
         let roles = ['GA', 'ANALYTICS', 'HELPDESK', 'SUPPORT']
         if (this.user && this.user.role && this.user.role.role_code) {
+<<<<<<< HEAD
```
Comment (sjrumsby) [inline frontend/src/layout/nav.vue]: What's this doing here?

File: frontend/src/main.js [hunk 2]
```diff
@@ -26,6 +26,8 @@ import './assets/css/bc-gov-style.css'
 import { library } from '@fortawesome/fontawesome-svg-core'
 import {
   faSort,
+  faFilter,
```
Comment (sjrumsby) [inline frontend/src/main.js]: Do these imports need to be in-lined? As in are they too long for the linter? They should probably be alphabetized as well

File: frontend/src/router.js [hunk 3]
```diff
@@ -22,41 +22,52 @@ import ButtonsDash from '@/serve-citizen/dash-buttons'
...
-      path: '/',
-      component: App,
-      redirect: '/queue',
-      children: [
-        {
-          path: 'queue',
-          components: {
-            default: Dash,
-            buttons: ButtonsDash,
+    mode: 'history',
```
Comment (sjrumsby) [inline frontend/src/router.js]: What's with all the white space change here?

**Comment by scottrumsby** [comment]
no idea about the whitespace changes... the files  look correct as they stand now..  Spaces, not tabs, 2 of them, maximum one blank line anywhere, no spaces on such a line and no extra spaces at the end of a line.

I don't think i can really do anything about this can I?  I'm assuming there are changes being noted in git because it must have had some of the above things going on previosuly

**Pull request merged**